slack_sdk/http_retry/builtin_handlers.py

```diff
@@ -39,6 +39,10 @@ def _can_retry(
         if error is None:
             return False
 
+        if isinstance(error, URLError):
```

Comment:
As HttpError is a sub-class of URLError, we should have checked if the `response` exists here.